AssetRipper.Fundamentals/Configuration/ExportDependencies.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AssetRipper.Core.Configuration
+{
+	public enum ExportDependencies
+	{
+		/// <summary>
+		/// Dependencies are exported with the project.
+		/// </summary>
+		Export,
+		/// <summary>
+		/// Dependencies are ignored during export.
+		/// </summary>
+		Ignore,
+	}
+}
```

Comment:
This is not a setting I want exposed. I purposefully didn't include it in the GUI.

Reply:
Interesting, what's the reason for not exposing that? Seems like useful functionality.